src/constants.ts

```diff
@@ -60,3 +60,12 @@ export const BRIDGED_USDC_SYMBOLS = [
   TOKEN_SYMBOLS_MAP.USDbC.symbol,
   TOKEN_SYMBOLS_MAP.USDzC.symbol,
 ];
+
+export const CUSTOM_GAS_TOKENS = {
+  [CHAIN_IDs.POLYGON]: "MATIC",
+  [CHAIN_IDs.POLYGON_AMOY]: "MATIC",
+  [CHAIN_IDs.ALEPH_ZERO]: "AZERO",
+  // FIXME: Replace with GRASS price once listed on Coingecko.
+  // For testing purposes, we use ETH price instead.
+  [CHAIN_IDs.LENS_SEPOLIA]: "ETH",
```

Comment:
What are the implications of using eth here other than fee misleading fee calculation?

Reply:
I don't think there should be any other implications other than incorrect relayer fee calculation. Because this is the only place where we use this constant (currently at least)